### PR TITLE
Changes up the Construction Area on Box slightly.

### PR DESCRIPTION
**Pull request by praisenarsie**
This is just so you can connect to the powernet easily, also there's no vent in there so I added one. Hopefully mapdiff will show?

### CODE REVIEW

**Comment by francinum** [comment]
@praisenarsie needs a picture

**Comment by praisenarsie** [comment]
mapwiffbot threw away the pictures for some raisin

**Comment by Cyberboss** [comment]
Served a few bad requests and the github cdn is now caching the empty picture, should come back after some refreshes

**Comment by ShizCalev** [comment]
```
/obj/machinery/power/apc{
	pixel_y = -24
	},
```

Would you mind naming the APC?

**Comment by praisenarsie** [comment]
wtf i thought i named the APC, maybe fastdmm is broke 🤷‍

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 1]
```diff
@@ -54070,26 +54070,33 @@
...
 /turf/open/floor/plating,
 /area/construction)
-"vJu" = (
+"kSb" = (
+/obj/effect/turf_decal/bot,
+/turf/open/floor/plasteel,
+/area/quartermaster/miningdock)
+"nxv" = (
 /obj/machinery/power/apc{
+	name = Construction Area APC;
```
Comment (SpaceManiac) [inline _maps/map_files/BoxStation/BoxStation.dmm]: uh oh

**Comment by ShizCalev** [comment]
fuck

**Pull request merged**